sklearn/mixture/gmm.py

```diff
@@ -649,6 +649,23 @@ def aic(self, X):
         return - 2 * self.score(X).sum() + 2 * self._n_parameters()
 
 
+@deprecated("The class GMM is deprecated and "
+            "will be removed in 0.20. Use class GaussianMixture instead")
+class GMM(_GMMBase):
+    def __init__(self, n_components=1, covariance_type='diag',
+                 random_state=None, tol=1e-3, min_covar=1e-3,
+                 n_iter=100, n_init=1, params='wmc', init_params='wmc',
+                 verbose=0):
+        super(GMM, self).__init__(
+            n_components=n_components, covariance_type=covariance_type,
+            random_state=random_state, tol=tol, min_covar=min_covar,
+            n_iter=n_iter, n_init=n_init, params=params,
+            init_params=init_params, verbose=verbose)
+
+        warnings.warn(
```

Comment:
Isn't it redundant with the `@deprecated` decorator ?
If not, should we use  `warnings.warn(msg, category=DeprecationWarning)`?